pdns/dnsdistdist/dnsrulactions.cc

```diff
@@ -1,132 +0,0 @@
-/*
```

Comment:
You are moving dnsrulactions.cc from the dnsdistdist dir to the pdns one, while we try to keep files that are only used by dnsdist in the dnsdistdist directory. I know I haven't been very consistent about that in the past, sorry.